lib/chat_api_web/controllers/company_controller.ex

```diff
@@ -6,6 +6,19 @@ defmodule ChatApiWeb.CompanyController do
 
   action_fallback ChatApiWeb.FallbackController
 
+  plug :authorize when action not in [:index, :create]
```

Comment:
let's still with `in` vs `not in` for now, i think it makes it a bit more obvious
```suggestion
  plug :authorize when action in [:show, :update, :delete]
```